gateway/gateway.go

```diff
@@ -1336,6 +1338,9 @@ func (gateway *HandleT) StartWebHandler(ctx context.Context) error {
 	srvMux.HandleFunc("/v1/failed-events", gateway.fetchFailedEventsHandler).Methods("POST")
 	srvMux.HandleFunc("/v1/clear-failed-events", gateway.clearFailedEventsHandler).Methods("POST")
 
+	sAPI := sources.API{}
+	//rudder-sources new APIs
+	srvMux.PathPrefix("/workspaces").Handler(sAPI.Handler())
```

Comment:
what is the purpose of the `/workspaces` path prefix?

Reply:
that's a mistake. Modified it to `/v1/job-status`.